EnglishFilter/sections/general_extensions.txt

```diff
@@ -243,6 +243,8 @@ kat.*,kickass.*,kickass2.*,kickasstorrents.*,kat2.*,kattracker.*,thekat.*,thekic
 !
 !
 !
+! strcloud.in - popups
+strcloud.in#%#//scriptlet("prevent-window-open")
```

Comment:
Do you need this one? This `#%#//scriptlet("abort-current-inline-script", "String.fromCharCode", "/emZnbG9hZGVk|break;case \$\.|break;case \d{1}/")` is enough on my end.

Reply:
I will remove it.

Reply:
@zloyden 
Btw, the normal HTML Rule for this scriptlet (`#%#//scriptlet("abort-current-inline-script", "String.fromCharCode", "/emZnbG9hZGVk|break;case \$\.|break;case \d{1}/")`) isn't working with the `min-length` and `max-length`. If I remove it and craft it to something like this  `strcloud.*$$script[wildcard="*function*break;case*"]`, then it's working.

Reply:
I have the same. Please check `strcloud.*$$script[wildcard="*function*break;case*"][min-length="30000"][max-length="100000"]`. The script is deleted normally.

Reply:
@zloyden Should I override the current rule or add it under HTML?

Reply:
You can rewrite the rule for the pull request - it's ok here.
Please notice that you should change it in 2 places.
1. Comment section.
2. Actual rule.

Reply:
Changed.